otelcollector/VERSION

```diff
@@ -1 +1 @@
-6.10.1
+6.10.2
```

Comment:
hi kaveesh - can u also please put a note in release process md file to include toggle & handover ? you can do it post this PR merge as well. i will approve this PR and you can resolve this to unblock you.

Reply:
Sounds good, will create another PR for this update.